benchmarks/tool/compile_benchmarks.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+# This does the same thing as `compile_benchmarks.sh`, but compiles inputs in
+# parallel.
+#
+# $ time ./tool/compile_benchmarks.py
+# ...
+# ./tool/compile_benchmarks.py  130.47s user 19.48s system 1074% cpu 13.951 total
+#
+# $ time ./tool/compile_benchmarks.sh
+# ...
+# ./tool/compile_benchmarks.sh  91.14s user 20.08s system 137% cpu 1:20.97 total
+
+import os
```

Comment:
I think it would be good that this script supports selecting the things to compile (golem e.g. doesn't want to compile everything always).

For example, support:
```
% tool/compile_benchmarks.py --target=jit,aot,js bin/
% tool/compile_benchmarks.py --target=aot bin/decode_json.dart
```
Thereby allowing to filter the target we're compiling for and the test (the former golem would use)


Furthermore for golem specifically it would be good if this script support "compiling" to JIT as well. This would significantly simplify the setup. So a
```
% tool/compile_benchmarks.py --target=jit bin/decode_json.dart
```

would use `dart --snapshot-kind=kernel --snapshot=out/decode_json.dart.dill bin/decode_json.dart` that can then be run via `dart out/decode_json.dart.dill` (just as one would from source via `dart bin/decode_json.dart`)

Reply:
Done.